ui/components/app/srp-quiz-modal/QuizContent/QuizContent.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {
+  AlignItems,
+  BlockSize,
+  Display,
+  FlexDirection,
+  JustifyContent,
+  TextAlign,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { BUTTON_SIZES, Button, Text } from '../../../component-library';
+import Box from '../../../ui/box';
+import { IQuizInformationProps } from '../types';
+
+export default function QuizContent({
+  icon,
+  image,
+  content,
+  moreContent,
+  buttons,
+}: IQuizInformationProps) {
+  const t = useI18nContext();
+
+  return (
+    <>
+      {icon && (
+        <Box
+          display={Display.Flex}
+          flexDirection={FlexDirection.Row}
+          alignItems={AlignItems.center}
+          justifyContent={JustifyContent.center}
+        >
+          {icon}
+        </Box>
+      )}
+      {image && (
+        <img src={image} alt={t('srpSecurityQuizImgAlt')} width="100%" />
+      )}
+      <Text
+        variant={TextVariant.bodyLgMedium}
```

Comment:
```suggestion
        variant={TextVariant.bodyLg}
```
This looks like it should be using the TextVariant.headingLg